charts/kube-prometheus-stack/values.yaml

```
@@ -2024,7 +2024,7 @@ prometheusOperator:
 
     ## SampleLimit defines per-scrape limit on number of scraped samples that will be accepted.
     ##
-    sampleLimit: 100
+    sampleLimit: 1000
```

i would go for "0" so the old behaviour is restored.

```suggestion
    sampleLimit: 0
```

---

good idea, There really isn't enough need to limit